app/models/Project.js

```diff
@@ -0,0 +1,93 @@
+const mongoose = require('mongoose')
+const Schema = mongoose.Schema
+const validator = require('validator')
+
+const projectSchema = new Schema({
+  projectName: {
+    type: String,
+    required: true,
+    trim: true,
+    minlength: 3,
+    validate (projectName) {
+      if (validator.isEmpty(projectName)) {
+        throw new Error('Project name is required!')
+      }
+      if (!validator.isLength(projectName, { min: 3 })) {
+        throw new Error('Project name should be 3 characters long!')
+      }
+    }
+  },
+  description: {
+    short: {
+      type: String,
+      required: true,
+      trim: true,
+      minlength: 10,
+      validate (short) {
+        if (validator.isEmpty(short)) {
+          throw new Error('Short description for the project is required!')
+        }
+        if (!validator.isLength(short)) {
+          throw new Error('Short description should be min 10 characters long!')
+        }
+      }
+    },
+    long: {
+      type: String,
+      trim: true
+    }
+  },
+  image: {
+    type: Buffer,
+    contentType: String
```

Comment:
add one more field to this - `imgUrl` -> this will contain aws s3 path of the img

Reply:
done! Let me know if any more changes required!